server/models/alert.js

```diff
@@ -89,29 +91,33 @@ Alert.prototype.testBalance = function(balance) {
            (this.order === 'gt' && balance >= alertLimit);
 };
 
-Alert.prototype.formatOperationMessage = function(operation, accountName) {
+Alert.prototype.formatOperationMessage = function(operation, accountName, locale, accountCurrency) {
```

Comment:
Can you make that the locale be the first argument, please? The order of arguments seems random like this.